meetings/2022-06-30.md

```diff
@@ -0,0 +1,141 @@
+* CB: That's from the server out of the box. For me, it is fine to say i can discover a notification channel for a storage.
+* SC: When you want to listen to updates of a particular resource, and go through storage, it doesn't mean that you'll have a way of guessing/tricking the server to make it disclose anything about `foo/bar` (especially if you are not permitted to know its existence or description).
+* SC: There are related issues to notification data, such as what that description reveals. So, servers will need to be careful to not expose or leak information more than necessary to enable the feature.
+* SC: There are some aspects of privacy that we need to look at, e.g., what is revealed. 
+* SC: Permission on the other hand is covered by the Solid Protocol.
+* RG: Is there already an open issue?
+* SC: #58 touches on that and also look on #76.
+* SC: The original was `Updates-Via` header that reveals the websocket endpoint for subscription. The problem was that the websocket endpoint was not doing auth.
+* CB: That is what I am using for my Web Push demo ... 
+* SC: :) Well, you know what to do next.
+* RG: Why not, e.g., `OPTIONS` and unique endpoint URL?
+* SC: Possible but then that exposes the secret which could get cached or inspected by intermediary?
+* RG: Suppose as a client I get a unique URL in a response with a small expire time...
+* SC: Messes with cache for the resource, I believe.
+
+### Solid Notifications Protocol: Review notes
+URL: https://github.com/solid/notifications/issues/76
+
+* SC: Talked with Tim about it (some specifics) but I didn't process the notes/review yet.
+* CB: Just took a look at them, but have to look at the protocol document along side it.
+* SC: I'll proccess/comment and update ED.
+
+
+### Subscription Request Response
+URL: https://github.com/solid/notifications/pull/72
+
+* SC: Follows action of https://github.com/solid/notifications-panel/blob/main/meetings/2022-05-19.md#protocol-doesnt-mention-subscription-response re https://github.com/solid/notifications/issues/57 .
+* SC: Updated PR.
+* CB: Will review after meeting.
+
+
+### Add definitions for notification channel discovery and capabilities
+URL: https://github.com/solid/notifications/pull/70
+
+* SC: There was plenty of time to review. Any objections to merge?
+* SC: Okay, great.
+
+
+### Interchangeable use of notification channel type and subscription type
+URL: https://github.com/solid/notifications/issues/71
+
+* SC: I think we can/should review the modeling around this.
+* SC: I think this is a bug. Lets come back to it with responses...
+* SC: I'd like confirmation from the others before making changes.
+* SC: From the spec:
+
+```turtle
+  <>
+    notify:notificationChannel <#websocketNotification> .
+
+  <#websocketNotification>
+    a notify:WebSocketSubscription2021 ;
+    notify:subscription <https://websocket.example/subscription> ;
+    notify:features notify:state, notify:rate, notify:expiration .
+```
```

Comment:
GitHub can't handle the triple-backtick within a suggestion :-(

There should be two leading space chars on both the opening (line 126) and closing (line 134) codefences on the Turtle block, to make that block indent under the SC bullet within which it should fall...

Reply:
I've tried that and checked in preview. It mangles it further. I think the current 'looks' right. I may have misunderstood your suggestion but if you want to make a suggestion that fixed whatever appears to be broken now, I can consider/merge.

Reply:
I think https://github.com/solid/notifications-panel/pull/62/commits/cafdd1df6822c4e95390bc25f506929e8a2ab99c worked. The preview through the GH editor is not the same as the view file. Let me know if that wraps it up. Thanks for looking into this!

Reply:
Looks good to me!